vmdb/config/vmdb.tmpl.yml

```diff
@@ -70,11 +70,15 @@ ems_refresh:
     get_shared_images: true
     get_public_images: false
     ignore_terminated_instances: true
-  scvmm:
+  foreman_configuration:
     :refresh_interval: 15.minutes
+  foreman_provisioning:
+    :refresh_interval: 1.hour
```

Comment:
I think the only reason to have it more frequently is so that we pick up new provisioning things that haven't been used by any configured system yet.